js/id/renderer/mapillary_sign_layer.js

```diff
@@ -9,9 +9,20 @@ iD.MapillarySignLayer = function(context) {
         currentImage,
         svg, image_preview_div, request, signs_defs;
 
-    request = d3.json('css/traffico-release-0.1.5/global-patched.json', function(error, data) {
+    var signs_defs={};
+    d3.json('css/traffico-release-0.1.17/string-maps/de-map.json', function(error, data) {
         if (error) return;
-        signs_defs = data;
+        signs_defs['eu'] = data;
```

Comment:
lint error `15:9   error  ["eu"] is better written in dot notation  dot-notation`